spec/models/ability_spec.rb

```diff
@@ -7,21 +7,44 @@
   subject { described_class.new(user) }
 
   let!(:wishlist) { create(:wishlist) }
+  let!(:suggestions) { create(:suggestions) }
   let(:user) { nil }
 
   it "has serializable permissions" do
     expect(subject.permissions).to include :can, :cannot
   end
 
-  context "when the user is a guest" do
+  context "when the user is a user" do
```

Comment:
The phrasing of this context feels unclear to me.

Reply:
let's call it `stranger`

Reply:
should I rename it stranger or it was a figure of speech

Reply:
yes please rename it to `stranger` in line 17 of this file 🙂 when you get the chance